### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by leonardoeng13**
Mobile Version Skeleton

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="desktop-nav">
+        <div class="menu-icon">
+            <img src="images/ic_menu.png" alt="">
+        </div>
+        <div class="logo-mobile">
+            <img src="images/image_geometry_m_index.png" alt="">
```
Comment (arslanbisharat) [inline index.html:19]: - [ ] it should be a link using a tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        </div>
+        <ul class="menu-items">
+            <li><a href="index.html">Hello</a></li>
+            <li><a href="Portfolio">Portfolio</a></li>
+            <li><a href="About">About</a></li>
+            <li><a href="Contact">Contact</a></li>
+        </ul>
```
Comment (arslanbisharat) [inline index.html:26]: - [ ] menu should be enclosed in nav tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="desktop-nav">
```
Comment (arslanbisharat) [inline index.html:14]: - [ ] this section should be enclosed in header tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            <img src="images/image_geometry_m_index.png" alt="">
+        </div>
+        <ul class="menu-items">
+            <li><a href="index.html">Hello</a></li>
+            <li><a href="Portfolio">Portfolio</a></li>
+            <li><a href="About">About</a></li>
+            <li><a href="Contact">Contact</a></li>
+        </ul>
+    </nav>
+    <header class="webcontainer">
```
Comment (arslanbisharat) [inline index.html:28]: - [ ] This section should be enclosed in section tag

File: index.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    </header>
+    <div class="imgindexmobile">
+        <img src="images/image_geometry_2_mobile.png" alt="">
+    </div>
+    <aside>
+        <div class="right">
+            <img src="images/image_geometry_1_1.png" alt="image_geometry_1_1.png">
+        </div>
+        <div class="dots">
+            <img src="images/image_dots.png" alt="image_dots.png">
```
Comment (arslanbisharat) [inline index.html:48]: - [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)
The button (if the template has one) is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design)

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limted number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (leonardoeng13) [reply index.html:19]: I believe this is the next step, now it's just the skeleton. I took some advantage and write more information to rush my work.

File: index.html (hunk 5 above)
Comment (leonardoeng13) [reply index.html:48]: I believe this is the next step, now it's just the skeleton. I took some advantage and write more information to rush my work.

File: index.html (hunk 2 above)
Comment (leonardoeng13) [reply index.html:26]: Semantic also not necessary at this point. Of course, it's a good way to fix this concept for me, but the primary is to prepare the skeleton.

File: index.html (hunk 3 above)
Comment (leonardoeng13) [reply index.html:14]: Semantic also not necessary at this point. Of course, it's a good way to fix this concept for me, but the primary is to prepare the skeleton.

File: index.html (hunk 4 above)
Comment (leonardoeng13) [reply index.html:28]: Semantic also not necessary at this point. Of course, it's a good way to fix this concept for me, but the primary is to prepare the skeleton.

File: index.html (hunk 1 above)
Comment (arslanbisharat) [reply index.html:19]: I am sorry but that is a requirement

File: index.html (hunk 2 above)
Comment (arslanbisharat) [reply index.html:26]: I am sorry but that is a requirement please make the changes accordingly

File: index.html (hunk 4 above)
Comment (arslanbisharat) [reply index.html:28]: I am sorry but that is a requirement

File: index.html (hunk 5 above)
Comment (arslanbisharat) [reply index.html:48]: I am sorry but that is a requirement

**Review by Shaher-11 — APPROVED** [review]
#### Hello @leonardoeng13,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Comment by Shaher-11** [comment]
Hi @leonardoeng13,
Although the project is approved I do recommend implementing the changes asked by the previous reviewer, as that will be helpful throughout the project.

**Pull request merged**